### DISCUSSION TITLE
Can someone help me delete the following account?

### DISCUSSION THREAD

**Discussion by 1NF053C** [community/community#136808] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

This account with Rust 101 beginner repos is no longer in use and I don't have access to it: https://github.com/darrow-olykos

**Comment by 1NF053C** [comment]
I can't submit a ticket on GitHub Support on free tier and I don't have access to the account any more -- it's nbd but there is also some code that is copied from a book that maybe shouldn't be published publicly so perhaps you could take it down for copyright reasons

**Reply by ebndev** [reply]
Hi @1NF053C,

GitHub Support will not able to accommodate this request.

For the future, please use [this link](https://support.github.com/) to submit tickets - as you are still able to as a user on the free tier.

Thank you!
